src/Profile.php

```diff
@@ -37,12 +36,12 @@ public function tokens()
     }
 
     /**
-     * @param  Request  $request
+     * @param  $request
      * @return AccessToken
      * @throws AccessTokenRequestException
      * @throws RequestTokenException
      */
-    public function requestToken(Request $request)
+    public function requestToken($request)
```

Comment:
If you really do want this to be type-hinted, we could improve the test to make it work. Up to you.

Reply:
in order to do it in this way, tests should be heavy refactored and then I'm think the actually signature of method `\Dymantic\InstagramFeed\Profile::requestToken` stands for poor code design. So, for now, we can let it intact.